lib/channel.rb

```diff
@@ -0,0 +1,30 @@
+class Channel < Recipient
+  attr_reader :topic, :member_count
+  
+  def initialize(topic: , member_count: , slack_id: , name: )
+    super(slack_id: slack_id, name: name)
+    @topic = topic
+    @member_count = member_count
+  end
+  
+  def details
+    details = """
```

Comment:
Triple quotes aren't needed in Ruby like they are in some other languages.  Ruby does multi-line strings by default.

```suggestion
    details = "
```

(In fact Ruby actually reads this as a empty string followed by a multi-line string followed by another empty string.  By default in Ruby strings get joined together if they are next to each other. `greeting = "Hello," " world!"` is the same as `greeting = "Hello, world!"`.)